lib/mixlib/install/script_generator.rb

```diff
@@ -195,10 +195,11 @@ def windows_metadata_url
         base = if omnibus_url =~ %r{/install.sh$}
                  "#{File.dirname(omnibus_url)}/"
                end
+        msi_version = %w{latest true nightlies}.include?(version) ? "" : version
 
         url = "#{base}#{endpoint}"
         url << "?p=windows&m=x86_64&pv=2008r2" # same package for all versions
-        url << "&v=#{CGI.escape(version.to_s.downcase)}"
+        url << "&v=#{CGI.escape(msi_version.to_s.downcase)}"
```

Comment:
Would this go cleaner if you just append `unless %w{latest true nightlies}.include?(version)` to this line? Then no need for new variable or indirection, and you don't pass a parameter you don't mean to pass.